sqlx-core/src/postgres/types/str.rs

```diff
@@ -9,13 +9,13 @@ use crate::Postgres;
 
 impl HasSqlType<str> for Postgres {
     fn type_info() -> PgTypeInfo {
-        PgTypeInfo::new(TypeId::TEXT)
+        PgTypeInfo::new(TypeId::TEXT, "TEXT")
     }
 }
 
 impl HasSqlType<[&'_ str]> for Postgres {
     fn type_info() -> PgTypeInfo {
-        PgTypeInfo::new(TypeId::ARRAY_TEXT)
+        PgTypeInfo::new(TypeId::ARRAY_TEXT, "TEXT")
```

Comment:
```suggestion
        PgTypeInfo::new(TypeId::ARRAY_TEXT, "TEXT[]")
```
Pretty sure. Just ignore if I'm wrong (it's late for me right now 😴 )

Reply:
Yep, thanks.